src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+
+    @Override
+    public List<? extends Machine> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        Bulldozer bull = new Bulldozer();
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
```suggestion
        Bulldozer firstBulldozer= new Bulldozer();
        Bulldozer secondBulldozer = new Bulldozer();
```

please pay more attention to naming